single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/SldParametersBuilder.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.sld;
+
+import com.powsybl.sld.layout.*;
+import com.powsybl.sld.library.ComponentLibrary;
+import com.powsybl.sld.library.ConvergenceComponentLibrary;
+import com.powsybl.sld.svg.DefaultLabelProvider;
+import com.powsybl.sld.svg.LabelProviderFactory;
+import com.powsybl.sld.svg.SvgParameters;
+import com.powsybl.sld.svg.styles.DefaultStyleProviderFactory;
+import com.powsybl.sld.svg.styles.StyleProviderFactory;
+
+import java.util.Objects;
+
+/**
+ *
+ * @author Sophie Frasnedo <sophie.frasnedo at rte-france.com>
+ */
+
+public class SldParametersBuilder {
```

Comment:
Same remark that for NadParameterBuilder. This is over-engineered. A simple SldParameters mutable bean with pre-initialized values would be a much simpler and user friendly API.
```java
new SldParameters()
    .setSvgParameters(new SvgParameters(...))
```

Reply:
Done :)